docker-entrypoint.sh

```diff
@@ -10,6 +10,6 @@ fi
 
 cp /etc/nginx/conf.d/default.conf /etc/nginx/conf.d/default.conf.template
 
-envsubst '$RESOLVERS $UPSTREAM_HTTP_ADDRESS $CLIENT_MAX_BODY_SIZE' < /etc/nginx/conf.d/default.conf.template > /etc/nginx/conf.d/default.conf
+envsubst '$RESOLVERS $UPSTREAM_HTTP_ADDRESS $CLIENT_MAX_BODY_SIZE $PROXY_REDIRECT' < /etc/nginx/conf.d/default.conf.template > /etc/nginx/conf.d/default.conf
```

Comment:
You need to have a default value of `default` for this. Otherwise, for the older apps where they don't have this new variable as their env var, the directive will be empty, hence incorrect. 

The default value for `proxy_redirect` is `default` as per nginx's docs http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_redirect